src/core_modules/capture-core/components/DataEntries/TeiRegistrationEntry/TeiRegistrationEntry.component.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import { useScopeInfo } from '../../../hooks/useScopeInfo';
+import { scopeTypes } from '../../../metaData';
+import { TrackedEntityInstanceDataEntry } from '../TrackedEntityInstance';
+import { useCurrentOrgUnitInfo } from '../../../hooks/useCurrentOrgUnitInfo';
+import type { OwnProps } from './TeiRegistrationEntry.types';
+import { useRegistrationFormInfoForSelectedScope } from '../common/useRegistrationFormInfoForSelectedScope';
+
+
+export const TeiRegistrationEntryComponent = ({ selectedScopeId, id, ...rest }: OwnProps) => {
```

Comment:
`...rest` is fine, but I suggest we stick to `passOnProps` like the rest of the app.

Reply:
You right there isnt any reason from my side to  changed it.